index.html

```diff
@@ -399,6 +399,34 @@ <h3>Behavior of a MediaStreamRecorder</h3>
       </p>
     </section>
   </section>
+  <section>
+    <h2>Security and Privacy Considerations</h2>
+    <p>
+      This specification adds an API where the user can send
+      information to the platform. This information is not
+      communicated elsewhere, and is not stored in any permanent
+      location.
+    </p>
+    <p>
+      By probing this API, the client may get some information on how
+      the platform implements this specification, which may give some
+      insight into its configuration. Apart from this, no platform
```

Comment:
Should it say "User Agent" rather than "platform"? Otherwise, can you clarify what kind of information we refer to when we say "insight into its configuration"?